tiledb/sm/misc/uri.cc

```diff
@@ -110,12 +110,24 @@ bool URI::is_invalid() const {
 }
 
 bool URI::is_file(const std::string& path) {
+#ifdef _WIN32
+  return utils::parse::starts_with(path, "file://") ||
+         path.find("://") == std::string::npos;
+#else
   return utils::parse::starts_with(path, "file:///") ||
```

Comment:
Isn't this also valid on windows?

Reply:
Assuming your referring to the 3 slashes (empty contents), it also matches with one less '/',
"file://" will also match/accept
"file:///"